client/ayon_blender/plugins/create/create_workfile.py

```diff
@@ -86,15 +85,17 @@ def create(self):
             workfile_instance["task"] = task_name
             workfile_instance["productName"] = product_name
 
-        convert_avalon_containers()
-        instance_node = bpy.data.collections.get(AYON_CONTAINERS)
+        instance_node = bpy.data.collections.get(AYON_INSTANCES)
         if not instance_node:
-            instance_node = bpy.data.collections.new(name=AYON_CONTAINERS)
+            instance_node = bpy.data.collections.new(AYON_INSTANCES)
+            instance_node.color_tag = "COLOR_04"
+            instance_node.use_fake_user = True
+            bpy.context.scene.collection.children.link(instance_node)
+
         workfile_instance.transient_data["instance_node"] = instance_node
 
     def collect_instances(self):
-
-        instance_node = bpy.data.collections.get(AYON_CONTAINERS)
+        instance_node = bpy.data.collections.get(AYON_INSTANCES)
```

Comment:
This needs backwards compatibility? It should still be able to find the old data and move it to the new place perhaps?

If it can't find the new - try the old?